tests/integrations/test_dsm.py

```diff
@@ -11,14 +11,28 @@
 
 import kombu
 
+# Kafka specific
+DSM_CONSUMER_GROUP = "testgroup1"
+
+# RabbitMQ Specific
+DSM_EXCHANGE = "dsm-system-tests-exchange"
+DSM_ROUTING_KEY = "dsm-system-tests-routing-key"
+
+# AWS Kinesis Specific
+DSM_STREAM = "dsm-system-tests-stream"
+
+# Generic
+DSM_QUEUE = "dsm-system-tests-queue"
+DSM_TOPIC = "dsm-system-tests-topic"
+
 
 @features.datastreams_monitoring_support_for_kafka
 @scenarios.integrations
 class Test_DsmKafka:
     """ Verify DSM stats points for Kafka """
 
     def setup_dsm_kafka(self):
-        self.r = weblog.get("/dsm?integration=kafka")
+        self.r = weblog.get(f"/dsm?integration=kafka&queue={DSM_QUEUE}&group={DSM_CONSUMER_GROUP}")
```

Comment:
Are we running multiple tests in the same queue/exchange?  If so, are we sure we're reading the right message in the tests below?

If the above is an issue, we can use fixtures to generate queues/etc on a per-test basis; although I don't know how that works with "warmups" in system tests

Reply:
No, we are using different queues / exchanges for each test.